doc/sphinx/manual/event_extraction.rst

```diff
@@ -82,38 +82,39 @@ To view the isolated events in a new window, you have to switch to the event edi
 
 ::
 
     >>> import minidemo # if you have not imported it already
     >>> minidemo.final()
 
-The final template should look similar as shown in Fig. 25
+The final template should look similar as shown in Fig. 25 and give time constants of 0.13 and 15.36 ms respectively.
 
 
     .. figure:: images/finaltemplate.png
         :align: center
 
         **Fig. 25:** Creating a final template.
 
-Extract all events
-------------------
+4. Extract all events
+---------------------
 
-Go back to the original file (minis.dat). Extracting all events with the final template is done in nearly the same way as described above for the preliminary template. However, you have to choose the correct template in the event dialog: The final template in this case is the second on the list (Fig. 26). For this final run, we will lower the detection threshold to a value of 3, as suggested by Clements and Bekkers (1997).
+In the original file (minis.dat) we will extract all events with the final template. As before, select in Analysis -> Event detection -> Template matching, but now the final template is the second on the menu list (Fig. 26). For this final run, we will lower the detection threshold to a value of 3, as suggested by Clements and Bekkers (1997).
 
 
     .. figure:: images/selectfinaltemplate.png
         :align: center
 
         **Fig. 26:** Selecting the final template.
 
-Edit detected events
---------------------
-Usually, the detected events have to be screened visually to remove false-positives and add false-negatives. Removing false-positives is done by unselected the checkbox next to the arrow indicating an event (Fig. 23). To add false-negatives, you have to switch to the event-editing mode (Fig. 24) and then right-click on the trace at the at the position where the event starts. from the context menu that will pop up, select "Add an event that starts here" (Fig. 27). To efficiently screen the whole trace, it is convenient to use **Shift**  and left arrow at the same time. this will move the trace left by the width of one window. Once you are done with editing, choose "Extract selected events" from the context menu.
+5. Edit detected events
+-----------------------
+
+The detected events have to be visually inspected to remove false-positives and add false-negatives. Remove false-positives with the unselected the checkbox next to the arrow indicating an event (Fig. 23). To add false-negatives, switch to the event-editing mode (Fig. 24) and right-click on the trace at the position where the event starts (Fig. 27). To screen the whole trace, it is convenient to use **Shift**  and left arrow at the same time. Once you are done with editing, choose "Extract selected events" from the context menu.
```

Comment:
"Remove false-positives with the unselected the checkbox next to the arrow indicating an event", please correct